boot-core/src/daemon/core.rs

```diff
@@ -220,6 +220,13 @@ impl TxHandler for Daemon {
         Ok(())
     }
 
+    fn wait_time(&self, secs: u64) -> Result<(), DaemonError> {
```

Comment:
@CyberHoward think wait_seconds might be clearer to the dev? Ohterwise lgtm

Reply:
renamed to wait_seconds